src/components/ManaCalculator/components/OutputForm.tsx

```diff
@@ -4,7 +4,15 @@ import { fromMicro } from '../utils';
 
 export function OutputForm() {
   const { state } = useManaState();
-  const { manaGenerated, passiveRewards, totalTPS } = useResults(state);
+  const {
+    manaGenerated,
+    passiveRewards,
+    totalTPS,
+    timeToGetEnoughMana,
+    timeToGetEnoughManaAsHolder,
+  } = useResults(state);
+  const humanizeDuration = require('humanize-duration');
```

Comment:
You could import it like a normal import too `import humanizeDuration from 'humanize-duration';`

Reply:
Nevermind it was a caching Problem of vs code 😅